src/MouseAdapter.js

```diff
@@ -309,20 +383,196 @@ define(function(require) {
         _onMouseWheel: function(event) {
             // If we haven't started a mouse wheel for a bit, publish a start event.
             if (!this._wheeling) {
-                this.onMouseWheelStart.dispatch([{
-                    distance: { x: 0, y: 0 },
-                    source: event
-                }]);
-                this._wheeling = true;
+                this._dispatchMouseWheelStart(event);
             }
 
             // Normalize the mouse wheel event and publish.
             var normalizedEvent = normalizeEvent(event);
+            this._detectUserScroll(normalizedEvent);
             this.onMouseWheel.dispatch([normalizedEvent]);
 
             // Debounce mouse wheels so that an end event is published 50ms
```

Comment:
This is now 90ms. Would be good to have these values as constants at the top of the file, as well as a few comments as to why their values were chosen (e.g. min value that safari supports, or something like that).